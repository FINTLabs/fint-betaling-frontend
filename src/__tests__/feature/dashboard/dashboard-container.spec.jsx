import renderer from 'react-test-renderer';
import React from 'react';
import DashboardContainer from '../../../feature/dashboard/dashboard-container';

jest.mock('../../../feature/dashboard/card-menu', () => () => 'cardmenu');
jest.mock('../../../feature/dashboard/payment_information/payment_information_list', () => () => 'paymentlist');

describe('Dashboard container', () => {
    it('should match the snapshot', () => {
        const tree = renderer.create(<DashboardContainer />)
            .toJSON();

        expect(tree)
            .toMatchSnapshot();
    });
});
