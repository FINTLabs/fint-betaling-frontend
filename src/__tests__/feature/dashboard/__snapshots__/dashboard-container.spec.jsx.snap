// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard container should match the snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-3 makeStyles-root-1"
>
  cardmenu
</div>
`;
